### PR TITLE
feat(login): WalletConnect v2 migration

### PR DESCRIPTION
**Pull request by Todmy**
### Issues
#3385 

### Changes 
*This PR is only for testing migration, which was done in `lock` library*

### How to test
1. Open the page
2. Login with Wallet Connect
3. Switch wallets
4. Try to perform an action (Vote, Proposal creation, etc)
5. Log out with Wallet Connect

### To-Do
- [ ] 

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [x] I have tested my changes on different screen sizes (sm, md)
- [x] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### CODE REVIEW

**Comment by samuveth** [comment]
Have you tried connecting with any wallet using the QR code. For me MetaMask and Zerion didn't work. Maybe we need to wait until the 28th.

Can you resolve the conflicts?

File: src/helpers/connectors.json
Comment (Xbjb8rVZ7ZJquS96Y4AJtNE4Pe9eNY1QGv) [inline src/helpers/connectors.json:1]: src/helpers/connectors.json

File: package.json
Comment (Xbjb8rVZ7ZJquS96Y4AJtNE4Pe9eNY1QGv) [inline package.json:1]: package.json

**Review by Xbjb8rVZ7ZJquS96Y4AJtNE4Pe9eNY1QGv — CHANGES_REQUESTED** [review]

**Review by ChaituVR — APPROVED** [review]
tAck

**Comment by aurelianoB** [comment]
Tried connecting to MM mobile with QR code without succcess

**Pull request merged**